junebug/workers.py

```diff
@@ -89,6 +88,24 @@ def store_and_forward_event(self, event):
         URL.'''
         yield self._store_event(event)
         yield self._forward_event(event)
+        yield self._count_event(event)
+
+    def _increment_metric(self, label):
+        return self.message_rate.increment(
+            self.channel_id, label, self.config['metric_window'])
+
+    def _count_event(self, event):
+        if event['event_type'] == 'ack':
+            return self._increment_metric('submitted')
+        if event['event_type'] == 'nack':
```

Comment:
What would be the advantage? Since we're returning for each branch of the if statements?